tests/figures/chart/charts_component.test.ts

```diff
@@ -1669,7 +1737,11 @@ describe("charts", () => {
           model,
           {
             dataSets: [
-              { dataRange: "B1:B4", trend: { type: "polynomial", order: 3, display: true } },
+              {
+                dataRange: "B1:B4",
+                label: "serie_1",
```

Comment:
nitpick: there should be a default value that can be used in the `await setInputValueAndTrigger(".data-series-selector" ...)` without having to manually set a label to each dataset at each test no ? That'd remove half the changes you had to make in the tests